lib/pages/classes/model/class.dart

```diff
@@ -54,11 +55,13 @@ class Class {
       {@required this.header,
       List<Shortcut> shortcuts,
       this.grading,
-      this.gradingLastUpdated})
+      this.gradingLastUpdated,
+      this.lecturer})
       : shortcuts = shortcuts ?? [];
 
   ClassHeader header;
   final List<Shortcut> shortcuts;
   final Map<String, double> grading;
   final LocalDateTime gradingLastUpdated;
+  final Person lecturer;
```

Comment:
I wouldn't save this as a property of class, we can just grab it dynamically from events.